src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -46,19 +46,19 @@ public void test_nameContainsKeywords_returnsTrue() {
         // One keyword
         NameContainsKeywordsPredicateOr predicate =
                 new NameContainsKeywordsPredicateOr(Collections.singletonList("Alice"));
-        assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
         // Multiple keywords
         predicate = new NameContainsKeywordsPredicateOr(Arrays.asList("Alice", "Bob"));
-        assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
         // Only one matching keyword
         predicate = new NameContainsKeywordsPredicateOr(Arrays.asList("Bob", "Carol"));
-        assertTrue(predicate.test(new PersonBuilder().withName("Alice Carol").build()));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Carol").build()));
 
         // Mixed-case keywords
         predicate = new NameContainsKeywordsPredicateOr(Arrays.asList("aLIce", "bOB"));
-        assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
```

Comment:
Instead of changing to assertFalse maybe you can do:
`
predicate = new NameContainsKeywordsPredicateOr(Arrays.asList("aLIce"));
assertTrue(predicate.test(new PersonBuilder().withName("Alice").build()));
`